src/platforms/javascript/index.mdx

```diff
@@ -26,63 +26,49 @@ $ yarn add @sentry/browser
 $ npm install --save @sentry/browser
 ```
 
-```html {tabTitle: CDN}
-<script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
-  crossorigin="anonymous"
-></script>
-```
+<Alert>
+<markdown>
 
-**Use of Defer**
+If you are using CDN to install our SDK, please see our documentation for [installing with CDN](/platforms/javascript/configuration/install-cdn/).
 
-It's possible to include `defer` in your script tag, but keep in mind that any errors which occur in scripts that execute before the browser SDK script executes won’t be caught (because the SDK won’t be initialized yet). We strongly recommend that if you use `defer`, you a) place the script tag for the browser SDK first, and b) mark it, and all of your other scripts, `defer` (but not `async`), thereby guaranteeing that it’s executed before any of the others.
+If you have Content Security Policy (CSP) set up on your site, please see [our documentation about adding to the `script-src`](/platforms/javascript/troubleshooting/).
```

Comment:
I believe this only applies if you're using the CDN, so now that the CDN stuff is on its own page, I think this can disappear from here.

Reply:
agreed. I thought I had removed it. Thanks for the catch.

Reply:
Changed this to "If you are using CDN to install our SDK, please see our documentation for [installing with CDN](/platforms/javascript/configuration/install-cdn/)"

Reply:
But it's still here...

Reply:
Just pushed the final changes :)